string-manipulation-1/src/is-lower-cased.ts

```diff
@@ -0,0 +1,7 @@
+/* exported isLowerCased */
+function isLowerCased(word: string): boolean {
+  if (typeof word !== 'string') {
+    return false;
+  }
```

Comment:
It's good to see you using guard clauses to try and stop certain inputs, but you don't need to do this check because you have TypeScript for that! You've already typed the parameter to be of type string so we can only accept strings as arguments when we call this function.